app/screens/Info/InfoView.js

```diff
@@ -6,7 +6,7 @@ export default class InfoView extends React.Component {
   render() {
     return (
       <View>
-        <Text>Info of {this.props.navigation.getParam('place')}</Text>
+        <Text>Info of {this.props.navigation.getParam('place').title}</Text>
```

Comment:
avoid extracting the data on view rendering process. 
Could use sth like `const title = this.props.navigation.getParam('place').title;` in render function and use as `<Text>Info of {title}</Text>`